### PR TITLE
Gsoc 2023 Juliastats XKDR proposals

### PR DESCRIPTION
**Pull request by smishr**
Adding the PR for GSoc JuliaStats and XKDR proposals.

Currently work in progress, we are compiling and adding XKDR proposals.

### CODE REVIEW

File: jsoc/gsoc/juliastats.md [hunk 1]
```diff
@@ -140,3 +140,46 @@ Participants will:
...
+-   Knowledge of basic statistical and probability concepts, preferably covered from academic course(s).
+-   Knowledge of basic software engineering practices, structured/object oriented programming.
+-   Basic familiarity or willingness to learn applied survey sampling and analysis. (Can be learned after applying. Mentorship and guidance provided.)
+-   (Bonus) Any prior experience using Julia for data analytics or statistics.
+-   (Bonus) Any prior experience or coursework with survey analysis, using any software or tool.
+-   (Bonus) Any prior experience with version control, git/GitHub, and contributing/PR to open-source projects.
+
+### Your contribution
+
+Internship can be tailored around brackground and interests of participants and depending on ability, several standalone mini-projects can be created. Potential participants can expect to work on:
```
Comment (aviks) [inline jsoc/gsoc/juliastats.md:165]:
```suggestion
The project can be tailored around the background and interests of participants and depending on ability, several standalone mini-projects can be created. Potential participants can expect to work on:
```

File: jsoc/gsoc/juliastats.md [hunk 2]
```diff
@@ -140,3 +140,46 @@ Participants will:
...
+-   Making Survey.jl better! This includes enhancing existing API with more functionality, new features and algorithms, software engineering improvements or anything else you're interested in.
+-   Generalised variance estimation methods - taylor series linearisation, Horvitz-Thompson style estimators
```
Comment (aviks) [inline jsoc/gsoc/juliastats.md:167]:
```suggestion
-   Generalised variance estimation methods - Taylor series linearisation, Horvitz-Thompson style estimators
```

File: jsoc/gsoc/juliastats.md [hunk 3]
```diff
@@ -140,3 +140,46 @@ Participants will:
...
+-   Association tests (with contingency tables), Rao-Scott, likelihood ratio tests for glms, Cox models, loglinear models.
+-   Causal inference and bayesian survey methods.
+-   Handling missing data, imputation.
+-   Increase performance of methods, analysing complexity, benchmarking code profiling, optimisation of code and algorithms
+-   Enhance documentation and tutorials, improve testing suite
+
+### References
+
+-   [Survey.jl](https://github.com/xKDR/Survey.jl) - see some issues, past PR's and milestone ideas
+-   Package announcements, Julia Discourse post [link here]
```
Comment (aviks) [inline jsoc/gsoc/juliastats.md:181]: remove for now?

File: jsoc/gsoc/juliastats.md [hunk 4]
```diff
@@ -140,3 +140,46 @@ Participants will:
...
+-   Benchmarking [link] - if done in time
```
Comment (aviks) [inline jsoc/gsoc/juliastats.md:182]: remove?

**Comment by ayushpatnaikgit** [comment]
@ParadaCarleton

**Pull request merged**